packages/api/models/schemas.js

```diff
@@ -474,6 +474,10 @@ module.exports.rule = {
         type: 'string',
       },
     },
+    executionNamePrefix: {
```

Comment:
I'm curious: why not put it in rule.meta which automatically goes into $.meta on the workflow message?

Reply:
This value is used to set the execution name when the workflow is started. If the value were in the workflow's `$.meta`, it wouldn't be used.